cyclonedx-lib/build.xml

```diff
@@ -136,6 +142,34 @@
                 </jar>
         </target>
 
+        <property name="testSBOMFile" location="build/testSBOM.json"/>
+                  <delete file="${testSBOMFile}"/>
+                  <property name="privatePemFile" location="build/RSAPrivate.pem"/>
+                  <delete file="${privatePemFile}"/>
+        <property name="publicPemFile" location="build/RSAPublic.pub"/>
+                  <delete file="${publicPemFile}"/>
+
+        <target name="runSignAndVerifySBOM">
+                <exec command="openssl genrsa -des3 -passout pass:test -out ${privatePemFile} 2048"/>
+                <exec command="openssl rsa -in ${privatePemFile} -passin pass:test -pubout -out ${publicPemFile}"/>
+                <java classpath="${classpath}" classname="temurin.sbom.TemurinSignSBOM">
```

Comment:
These java targets need:
```
fork="yes"
failonerror="yes">
```
So that the ant task fails the build if the class fails.